e2e/basic/observability_test.go

```diff
@@ -73,15 +76,15 @@ scrape_configs:
 	s.Require().NoError(err)
 
 	s.Require().NoError(target.Sidecars().Add(ctx, observabilitySidecar))
+
 	s.Require().NoError(target.Build().Commit(ctx))
 	s.Require().NoError(target.Execution().Start(ctx))
 
 	// Wait for the target pod to push data to the otel collector
-	s.T().Log("Waiting one minute for the target pod to push data to the otel collector")
-	time.Sleep(1 * time.Minute)
+	s.T().Log("Waiting 20 seconds for the target pod to push data to the otel collector")
+	time.Sleep(20 * time.Second)
```

Comment:
Is there an opportunity here to use the `Require().Eventually()` to further reduce flakiness?

Reply:
will investigate it

Reply:
done, it's working now. thanks for pointing it out.